staging/src/k8s.io/apimachinery/pkg/conversion/converter.go

```diff
@@ -138,10 +138,6 @@ type Scope interface {
 	// parameters, you'll run out of stack space before anything useful happens.
 	Convert(src, dest interface{}, flags FieldMatchingFlags) error
 
-	// DefaultConvert performs the default conversion, without calling a conversion func
-	// on the current stack frame. This makes it safe to call from a conversion func.
-	DefaultConvert(src, dest interface{}, flags FieldMatchingFlags) error
```

Comment:
cc @smarterclayton for use downstream? I know you still want to drop reflect-based conversions and require everything to be generated... this seems like a decent first step

Reply:
I know there are places we use it but have a registered conversion.  I don’t know of a reason those places can’t just call it directly, and anyone else using direct conversion as a “copy but drop fields” is easily fixed to “copy explicit”.

Kill it.